components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -977,11 +977,15 @@ protected String sendRequest(String url, String accessToken) throws IOException
         return builder.toString();
     }
 
-    private String interpretQueryString(String queryString, Map<String, String[]> parameters) {
+    private String interpretQueryString(AuthenticationContext context, String queryString,
+                                        Map<String, String[]> parameters) {
 
         if (StringUtils.isBlank(queryString)) {
             return null;
         }
+        if (queryString.contains(OIDCAuthenticatorConstants.AUTH_PARAM)) {
+            return getQueryStringWithAuthenticatorParam(context, queryString);
```

Comment:
If we return here, We won't be able to use both types at the same time. Please refactor.
eg:
p1=${foo}&p2=$authparam{bar}

Reply:
Fixed in 6fa250926b42ac69bff4e7ea8c16bb97b5f636a3